packages/serverless-contracts/src/contracts/apiGateway/__tests__/fetchRequest.test.ts

```diff
@@ -95,6 +94,38 @@ describe('apiGateway fetch request', () => {
         },
       );
     });
+
+    it('should have the correct axiosRequest when some queryStringParameters are undefined', async () => {
+      await getFetchRequest(
+        httpApiContract,
+        mockedFetch as unknown as typeof fetch,
+        {
+          pathParameters: {
+            userId: 'azer',
+            pageNumber: 'zert',
+          },
+          queryStringParameters: {
+            testId: undefined,
+          },
```

Comment:
To better test the functionality, let's add another property with a defined value

Reply:
Just updated the test :)